src/main/java/ToDoServlet.java

```diff
@@ -0,0 +1,17 @@
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "ToDoServlet", value = "/hello")
```

Comment:
I guess you should decide whether you use annotation based approach or declaration servlets in web.xml